common/djangoapps/course_modes/models.py

```diff
@@ -608,6 +609,9 @@ def is_white_label(cls, course_id, modes_dict=None):
         if modes_dict is None:
             modes_dict = cls.modes_for_course_dict(course_id)
 
+        if configuration_helpers.is_site_configuration_enabled():
+            return configuration_helpers.get_value('show_ecommerce_reports', False)
```

Comment:
Okay, cool -- so @asadiqbal08 let's change the origin of the [reports_enabled](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/instructor/views/instructor_dashboard.py#L227) argument value being inserted [here](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/instructor/views/instructor_dashboard.py#L154), to use the new `show_ecommerce_reports` flag from SiteConfiguration instead of the result from `CourseMode.is_white_label()`.

Does that make sense?